modules/ducktests/tests/ignitetest/services/utils/ssl/connector_configuration.py

```diff
@@ -28,4 +28,4 @@ class ConnectorConfiguration(NamedTuple):
     Used to connect from ControlUtility (control.sh).
     """
     ssl_enabled: bool = False
-    ssl_context_factory: SslContextFactory = None
+    ssl_context: SslContext = None
```

Comment:
why get rid of the factory since it's the factory?

Reply:
It's not a factory, it just params really